### ISSUE TITLE
WALG_SSH_PREFIX not recognized as an option for Postgres

### ISSUE THREAD

**Issue by genslein** [wal-g/wal-g#735]
Hey all,

I've been trying to set up using SSH to retrieve backups for Postgres. It appears this is not an accepted option despite it being mentioned in documentation. We have a couple of different use cases for this so trying to understand how to set this up properly. Any guidance is appreciated. Currently using v0.2.15 from releases tab here in the project.

```
ERROR: 2020/08/21 12:27:07.502243 No storage is configured now, please set one of following settings: [WALG_S3_PREFIX WALG_FILE_PREFIX WALG_GS_PREFIX WALG_AZ_PREFIX WALG_SWIFT_PREFIX]
github.com/wal-g/wal-g/internal.newUnconfiguredStorageError
	/home/travis/gopath/src/github.com/wal-g/wal-g/internal/configure.go:43
github.com/wal-g/wal-g/internal.ConfigureFolder
	/home/travis/gopath/src/github.com/wal-g/wal-g/internal/configure.go:130
github.com/wal-g/wal-g/cmd/pg.glob..func1
	/home/travis/gopath/src/github.com/wal-g/wal-g/cmd/pg/backup_fetch.go:26
github.com/spf13/cobra.(*Command).execute
	/home/travis/gopath/src/github.com/wal-g/wal-g/vendor/github.com/spf13/cobra/command.go:830
github.com/spf13/cobra.(*Command).ExecuteC
	/home/travis/gopath/src/github.com/wal-g/wal-g/vendor/github.com/spf13/cobra/command.go:914
github.com/spf13/cobra.(*Command).Execute
	/home/travis/gopath/src/github.com/wal-g/wal-g/vendor/github.com/spf13/cobra/command.go:864
github.com/wal-g/wal-g/cmd/pg.Execute
	/home/travis/gopath/src/github.com/wal-g/wal-g/cmd/pg/pg.go:29
main.main
	/home/travis/gopath/src/github.com/wal-g/wal-g/main/pg/main.go:8
runtime.main
	/home/travis/.gimme/versions/go1.13.9.linux.amd64/src/runtime/proc.go:203
runtime.goexit
	/home/travis/.gimme/versions/go1.13.9.linux.amd64/src/runtime/asm_amd64.s:1357
```

**Comment by DhirenderYadav** [comment]
Do we need to add below parameter in postgres.conf file or any other file.

Can you please share the sample or example for SSH ?

To store backups via ssh, WAL-G requires that these variables be set:

WALG_SSH_PREFIX (e.g. ssh://localhost/walg-folder)
SSH_PORT ssh connection port
SSH_USERNAME connect with username
SSH_PASSWORD connect with password

**Comment by genslein** [comment]
@DhirenderYadav  you can use environment variables to execute the backup standalone. There's also an option to use a json file and pass `--config your_file_config.json` instead of doing embedded shell script.

```
#!/bin/bash

export PGHOST=localhost
export PGPORT=4342
export PGUSER=my_user
export PGPASSWORD="my_user_pass"

export PGDATA=/data/postgres/my_pg_data_dir
export WALG_UPLOAD_DISK_CONCURRENCY=3
export WALG_COMPRESSION_METHOD=lz4

# Supports S3 Like targets
export WALG_SSH_PREFIX=ssh://my-ipv4-dns-name:/walg-repo-folder
export SSH_PORT=22
export SSH_USERNAME=my_system_user
export SSH_PASSWORD=my_system_pass

# can add additional debug logging
export WALG_LOG_LEVEL=DEVEL

/my_postgres_executable_dir/wal-g backup-push $1
```

**Comment by genslein** [comment]
I think this is closed now `c2d98be4024db4a46b9bed56dd1809f9372d2b26`
